### ISSUE TITLE
Beta IDE v2024.200.0.486: Cannot open projects

### ISSUE THREAD

**Issue by Totobal5** [YoYoGames/GameMaker-Bugs#4663]
### Description
thats its
the projects atached can be opened in the stable branch, but when try to open it on the beta it appear the error message attached.

### Steps To Reproduce
1. Try to open the projects attached.

### Which version of GameMaker are you reporting this issue for?
IDE v2024.200.0.486 Runtime v2024.200.0.502

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

[4e9c810b-a868-47d4-a165-0056b01407af](https://api.gamemaker.io/api/github/downloads/4e9c810b-a868-47d4-a165-0056b01407af)

**Comment by stuckie** [comment]
Could we get a zip of the actual project which is failing to load? The ones provided appear to convert fine from what I've been looking at.
And just in case there's a slightly different path that's being followed, how, specifically, are you opening the project?
For example:
- Is it through the recent projects list?
- Are you dragging the YYP onto the start screen?
- Going through File->Open?
- Just double clicking and letting the file association open GameMaker with the project from the outset?

Additionally, when the project fails to load, if you could also attach the output from Windows -> Project Tool Output, that may help pin down what may be going on too.

**Comment by Totobal5** [comment]
> Could we get a zip of the actual project which is failing to load? The ones provided appear to convert fine from what I've been looking at. 
This is weird! every project i send has a problem when i try to load it!!

1) 
![GameMaker-Beta_sbRaJZpsp3](https://github.com/YoYoGames/GameMaker-Bugs/assets/90012149/ed924dd3-a842-4038-927e-28b7e8136fab)

this is the Ghost Seed project

```
ProjectTool Version 1.2
-----------------------
Checking 'VERSIONED' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\ghost-seed-main\Ghost Seed.yyp' - False/True : 170
Checking 'VERS0' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\ghost-seed-main\Ghost Seed.yyp' - False/False : 170
Checking 'MVC' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\ghost-seed-main\Ghost Seed.yyp' - False/False : 170
Checking 'GMX' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\ghost-seed-main\Ghost Seed.yyp' - False/False : 170
Setting up CoreResources for VERSIONED
Loading...C:/Users/xtoba/Documents/GameMakerStudio2/ghost-seed-main/Ghost Seed.yyp(2,3): GMSC Error: File parsing failed: Type tag missing when parsing slot of type YoYoStudio.Resources.ResourceBase
Core Resources : IDE Message - ProjectLoad_ErrorTitle : ProjectLoad_ErrorMessage ( C:/Users/xtoba/Documents/GameMakerStudio2/ghost-seed-main/Ghost Seed.yyp(2,3): GMSC Error: File parsing failed: Type tag missing when parsing slot of type YoYoStudio.Resources.ResourceBase )
Failed
Cannot load project at 'C:\Users\xtoba\Documents\GameMakerStudio2\ghost-seed-main\Ghost Seed.yyp' in format 'VERSIONED'
Cannot load project at: C:\Users\xtoba\Documents\GameMakerStudio2\ghost-seed-main\Ghost Seed.yyp
ProjectTool Failed
```


2)
![GameMaker-Beta_I10hVA6r5U](https://github.com/YoYoGames/GameMaker-Bugs/assets/90012149/ba7d1cfa-c5bc-446f-a23b-2810ea7d4798)

this is the Cueca project

```
ProjectTool Version 1.2
-----------------------
Checking 'VERSIONED' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Cueca\Cueca.yyp' - False/True : 170
Checking 'VERS0' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Cueca\Cueca.yyp' - False/False : 170
Checking 'MVC' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Cueca\Cueca.yyp' - False/False : 170
Checking 'GMX' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Cueca\Cueca.yyp' - False/False : 170
Setting up CoreResources for VERSIONED
Loading...C:/Users/xtoba/Documents/GameMakerStudio2/_Librerias_/Cueca/Cueca.yyp(2,3): GMSC Error: File parsing failed: Type tag missing when parsing slot of type YoYoStudio.Resources.ResourceBase
Core Resources : IDE Message - ProjectLoad_ErrorTitle : ProjectLoad_ErrorMessage ( C:/Users/xtoba/Documents/GameMakerStudio2/_Librerias_/Cueca/Cueca.yyp(2,3): GMSC Error: File parsing failed: Type tag missing when parsing slot of type YoYoStudio.Resources.ResourceBase )
Failed
Cannot load project at 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Cueca\Cueca.yyp' in format 'VERSIONED'
Cannot load project at: C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Cueca\Cueca.yyp
ProjectTool Failed
```


3)

![GameMaker-Beta_0kzVGUB7tN](https://github.com/YoYoGames/GameMaker-Bugs/assets/90012149/cab09aea-05b1-4853-bb1a-f5bd3e1112c4)

this is the Destructor project.
```

ProjectTool Version 1.2
-----------------------
Checking 'VERSIONED' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Destructors\Destructors.yyp' - False/True : 170
Checking 'VERS0' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Destructors\Destructors.yyp' - False/False : 170
Checking 'MVC' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Destructors\Destructors.yyp' - False/False : 170
Checking 'GMX' can load/import 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Destructors\Destructors.yyp' - False/False : 170
Setting up CoreResources for VERSIONED
Loading...C:/Users/xtoba/Documents/GameMakerStudio2/_Librerias_/Destructors/Destructors.yyp(2,3): GMSC Error: File parsing failed: Type tag missing when parsing slot of type YoYoStudio.Resources.ResourceBase
Core Resources : IDE Message - ProjectLoad_ErrorTitle : ProjectLoad_ErrorMessage ( C:/Users/xtoba/Documents/GameMakerStudio2/_Librerias_/Destructors/Destructors.yyp(2,3): GMSC Error: File parsing failed: Type tag missing when parsing slot of type YoYoStudio.Resources.ResourceBase )
Failed
Cannot load project at 'C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Destructors\Destructors.yyp' in format 'VERSIONED'
Cannot load project at: C:\Users\xtoba\Documents\GameMakerStudio2\_Librerias_\Destructors\Destructors.yyp
ProjectTool Failed
```


>And just in case there's a slightly different path that's being followed, how, specifically, are you opening the project? For example:

I open the projects just doing a double click on the file.

i hope this helps!!!

**Comment by stuckie** [comment]
This looks to be the same as #4664 
The project format number is correctly identified, but it hasn't actually converted it at all, and jumped straight to trying to verify it in the final format.

Thanks for that, I have a better idea of where to poke now.

**Comment by caitlinrmcintyre** [comment]
Verified fix in IDE v2024.200.0.494  Runtime v2024.200.0.511, thanks.
